### ISSUE TITLE
Adults won't move

### ISSUE THREAD

**Issue by LeP1ngas** [Luke100000/minecraft-comes-alive#908]
**Describe the bug**
The adult villagers do not move on their own, not even looking around or at the player. After giving the command to follow, then unfollowing, they regain their AI and start moving normally again. It includes the naturally spawing villigers and those spawnned from an egg. This does not apply to teenagers, they move as they should.

**To Reproduce**
There is nothing to do, just find a village or spawn some.

**Version**
minecraft-comes-alive-7.5.14+1.20.1-universal (from Modrinth), 1.20.1, Forge 47.1.30

**Latest.log**
[latest.log](https://github.com/Luke100000/minecraft-comes-alive/files/15078808/latest.log)

**Possible mod conflicts**
I MCA and Minecolonies together and thougt it might conflict with it. I also run way more mods but none affect the AI of villagers as far as I know.

**Comment by Luke100000** [comment]
Yes, sounds like an incompatibility with another mod, please find the conflicting one

**Comment by YTDoctor** [comment]
This is an incompatibility issue from what I've seen. I've been testing out the Forgified Fabric/Sinytra Connector mods to run Fabric mods on Forge, and running these along with MCA causes children to wander as normal, while adults stand still.

If it's not too big of an issue, you can ask the adults to follow you, then tell them to roam freely, and they'll wander/get jobs like normal. The issue doesn't entirely break the mod, just makes them stuck until you jostle them with an interaction.

EDIT**
On further examination, it doesn't even necessarily seem like ONLY adults are frozen. I've come across villages where some adults are not frozen, while others are. Strange bug, but not game breaking. Perhaps this is an easily solveable incompatibility? I've muddled around with the RAR files, but to no avail.

**Comment by Luke100000** [comment]
Not the cause of this specific issue, but Cyberchems as of now also disabled parts of the AI: https://github.com/Perigrine33/Cybernetics-Cyberchems/issues/2
